enterprise_catalog/apps/api/v1/views/enterprise_catalog_crud.py

```diff
@@ -80,6 +80,10 @@ def get_queryset(self):
         Returns the queryset corresponding to all catalogs the requesting user has access to.
         """
         all_catalogs = EnterpriseCatalog.objects.all().order_by('created')
+        customer_catalog = self.request.GET.get('enterprise_customer', False)
```

Comment:
I would rename this to `enterprise_customer` or `enterprise_uuid`

Reply:
I agree with you)